qutip/ui/progressbar.py

```diff
@@ -1,6 +1,6 @@
 __all__ = ['BaseProgressBar', 'TextProgressBar',
            'EnhancedTextProgressBar', 'TqdmProgressBar',
-           'progess_bars']
+           'progess_bars', 'HTMLProgressBar']
```

Comment:
```suggestion
           'HTMLProgressBar', 'progess_bars']
```
Could you move `HTMLProgressBar` before the list of `progress_bars` so that the items are neatly organized?

Reply:
@hodgestar Updated!